### PR TITLE
Add seven languages for BIP39 Wordlists

### PR DESCRIPTION
**Pull request by yuezy**

### CODE REVIEW

File: web3swift/KeystoreManager/Classes/BIP39.swift [hunk 1]
```diff
@@ -11,10 +11,31 @@ import CryptoSwift
 
 public enum BIP39Language {
     case english
+    case chinese_simplified
+    case chinese_traditional
+    case japanese
```
Comment (shamatar) [inline web3swift/KeystoreManager/Classes/BIP39.swift]: japanese language requires a special separator of seed words

Comment (skywinder) [reply web3swift/KeystoreManager/Classes/BIP39.swift]: @yuezy thank you for your addition. as soon as you add separators - we would be glad to merge! 👍

**Comment by skywinder** [comment]
Wordlists can be checked there:
https://github.com/bitcoin/bips/blob/master/bip-0039/bip-0039-wordlists.md

File: web3swift/KeystoreManager/Classes/BIP39.swift (hunk 1 above)
Comment (skywinder) [reply web3swift/KeystoreManager/Classes/BIP39.swift]: @yuezy we have to consider this in this commit:

>*Developers implementing phrase generation or checksum verification must separate words using ideographic spaces / accommodate users inputting ideographic spaces.* (UTF-8 bytes: 0xE38080; C/C+/Java: "\u3000"; Python: u"\u3000")
https://github.com/bitcoin/bips/blob/master/bip-0039/bip-0039-wordlists.md#japanese

File: web3swift/KeystoreManager/Classes/BIP39.swift [hunk 2]
```diff
@@ -11,10 +11,31 @@ import CryptoSwift
...
     var words: [String] {
         switch self {
         case .english:
             return englishWords
+        case .chinese_simplified:
+            return simplifiedchineseWords
+        case .chinese_traditional:
+            return traditionalchineseWords
+        case .japanese:
+            return japaneseWords
```
Comment (skywinder) [inline web3swift/KeystoreManager/Classes/BIP39.swift]: @yuezy we have to consider this in this commit:

>*Developers implementing phrase generation or checksum verification must separate words using ideographic spaces / accommodate users inputting ideographic spaces.* (UTF-8 bytes: 0xE38080; C/C+/Java: "\u3000"; Python: u"\u3000")

https://github.com/bitcoin/bips/blob/master/bip-0039/bip-0039-wordlists.md#japanese

**Comment by skywinder** [comment]
@yuezy  is there any updates on this case?

**Comment by shamatar** [comment]
Closing due to inactivity. Still appreciate the work, included by myself

**Pull request closed without merging**